source/BatchDrawList.h

```diff
@@ -60,7 +60,8 @@ class BatchDrawList {
 	// two dummy vertices to mark the break in between them). Each of those
 	// vertices has five attributes: (x, y) position in pixels, (s, t) texture
 	// coordinates, and the index of the sprite frame.
-	std::map<const Sprite *, std::vector<float>> data;
+	// Each Sprite has also an alpha value asigned.
+	std::map<const Sprite *, std::pair<std::vector<float>, double>> data;
```

Comment:
Can you make the pair an actual class (for readability as always)? thx

Reply:
you know I would love to make this a struct...
but I will make a class as always.

Reply:
done

Reply:
Oh, when I say class you can make it a struct too I don't mind.

Reply:
For some reason we don't do structs. Idk why.

Reply:
changed it to struct